packages/cli/tests/create-tutorial.test.ts

```diff
@@ -73,24 +73,25 @@ test('create and eject a project', async (context) => {
   const name = context.task.id;
   const dest = path.join(tmpDir, name);
 
-  await execa('node', [cli, 'create', name, '--no-git', '--no-start', '--defaults'], {
+  await execa('node', [cli, 'create', name, '--no-git', '--no-install', '--no-start', '--defaults'], {
     cwd: tmpDir,
-    env: {
-      TK_DIRECTORY: baseDir,
-    },
   });
 
+  await runPnpmInstall(dest, baseDir);
+
   await execa('node', [cli, 'eject', name, '--force', '--defaults'], {
     cwd: tmpDir,
-    env: {
-      TK_DIRECTORY: baseDir,
-    },
   });
 
-  // remove `node_modules` before taking the snapshot
-  await fs.rm(path.join(dest, 'node_modules'), { force: true, recursive: true });
+  if (process.platform !== 'win32') {
+    await fs.rm(path.join(dest, 'node_modules'), { force: true, recursive: true, maxRetries: 5 });
+  }
 
-  const projectFiles = await fs.readdir(dest, { recursive: true });
+  let projectFiles = await fs.readdir(dest, { recursive: true });
+
+  if (process.platform === 'win32') {
+    projectFiles = projectFiles.filter((filePath) => !filePath.startsWith('node_modules'));
+  }
```

Comment:
I really don't like this hack but I couldn't find any other better way to remove this folder on Windows.

It seems that the issue is that when we try to delete the `node_modules`, we also delete `esbuild.exe` which get an `EBUSY` error. If we use the `maxRetry` property for that usecase, it times out.

My guess is that at this point there's still an `esbuild` process running. However, I don't know what spawned that `esbuild` process and I don't want to try to randomly kill all `esbuild` processes which would be way worse.